### ISSUE TITLE
Feature Request: risc-v support

### ISSUE THREAD

**Issue by laoshaw** [embeddedartistry/libc#170]
can risc-v 32/64 support be added? Thanks!

**Comment by phillipjohnston** [comment]
Yeah, do you have any good RISC-V 32/64 development hardware recommendations?

**Comment by laoshaw** [comment]
I have a sifive unmatched board, I think to test qemu shall work fine.

**Comment by stefanct** [comment]
the sifive boards are expensive and the unmatched is very loud and not matured enough (at least some months ago - i temporarily gave up on it). i cannot recommend it and agree that most likely for libc development qemu is enough.

**Comment by laoshaw** [comment]
agree, for me I replaced the fan on sifive board and it's super quiet these days, but for libc development qemu should suffice.
